include-dependencies.js

```diff
@@ -53,17 +55,46 @@ module.exports = class IncludeDependencies {
     service.package.exclude = union(service.package.exclude, ['node_modules/**']);
 
     if (runtime.match(/nodejs*/)) {
+      this.processIncludes(functionObject);
       this.processNodeFunction(functionObject);
     }
   }
 
+  includeGlobs(target, include, exclude) {
+    include.forEach(includeGlob => {
+      glob.sync(path.join(this.serverless.config.servicePath, includeGlob))
+        .filter(p => !exclude.some(e => micromatch.contains(p, e)))
+        .forEach(filePath => {
+          var stat = fs.statSync(filePath);
+          if (stat && stat.isFile()) {
+            const dependencies = this.getDependencies(filePath, exclude);
+            this.include(target, dependencies);
+          }
+        });
+      }
+    );
+  }
+
+  processIncludes(functionObject) {
+    const service = this.serverless.service;
+    if (service.package && service.package.individually) {
+      const include = union(service.package.include, functionObject.package.include);
+      const exclude = union(service.package.exclude, functionObject.package.exclude);
+      this.includeGlobs(functionObject, include, exclude);
+    } else {
+      const include = service.package.include || [];
+      const exclude = service.package.exclude || [];
+      this.includeGlobs(service, include, exclude);
+    }
+  }
+
   processNodeFunction(functionObject) {
     const service = this.serverless.service;
 
     const fileName = this.getHandlerFilename(functionObject.handler);
 
     if (service.package && service.package.individually) {
-      const exclude = union(service.package.exclude, functionObject.exclude);
+      const exclude = union(service.package.exclude, functionObject.package.exclude);
```

Comment:
This may have been a bug?

Reply:
yes, you're right! but you need a guard against `functionObject.package` not being defined

Reply:
I added a test case, but it looks like `processFunction` ensures that `functionObject.package` is an object.